### PR TITLE
Update maintenane presets for devices from directories: Showtec, Sole…

### PR DESCRIPTION
**Pull request by kluszon**
The next part of fixtures with changed maintenance presets.

### CODE REVIEW

File: resources/fixtures/Showtec/Showtec-Pixel-Bar-12-RGBW.qxf [hunk 1]
```diff
@@ -129,8 +129,8 @@
  </Channel>
  <Channel Name="Linear Strobe">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="1">Strobe off</Capability>
-  <Capability Min="2" Max="255" Preset="StrobeFreqRange" Res1="0" Res2="20">Strobe from slow to fast</Capability>
+  <Capability Min="0" Max="1" Preset="ShutterOpen">Strobe off</Capability>
+  <Capability Min="2" Max="255" Preset="StrobeFastToSlow">Strobe from slow to fast</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Showtec/Showtec-Pixel-Bar-12-RGBW.qxf:133]: slow to fast

File: resources/fixtures/Showtec/Showtec-Acrobat.qxf [hunk 2]
```diff
@@ -18,51 +18,51 @@
...
  </Channel>
  <Channel Name="Shutter / Strobe">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="3">Closed / Blackout</Capability>
-  <Capability Min="4" Max="7">Shutter Open</Capability>
-  <Capability Min="8" Max="215">Strobe, Slow - Fast ( 0-10 Flash / Sec )</Capability>
-  <Capability Min="216" Max="255">Shutter Open</Capability>
+  <Capability Min="0" Max="3" Preset="ShutterClose">Closed / Blackout</Capability>
+  <Capability Min="4" Max="7" Preset="ShutterOpen">Shutter Open</Capability>
+  <Capability Min="8" Max="215" Preset="StrobeSlowToFast">Strobe, Slow - Fast ( 0-10 Flash / Sec )</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Showtec/Showtec-Acrobat.qxf:44]: I usually set it as StrobeFreqRange 0-10

File: resources/fixtures/Showtec/Showtec-Pixel-Bar-12-RGBW.qxf (hunk 1 above)
Comment (kluszon) [reply resources/fixtures/Showtec/Showtec-Pixel-Bar-12-RGBW.qxf]: Ok, sorry it's our mistake. Fixed.

File: resources/fixtures/Showtec/Showtec-Acrobat.qxf (hunk 2 above)
Comment (kluszon) [reply resources/fixtures/Showtec/Showtec-Acrobat.qxf:44]: Ok, should I change it?

**Comment by mcallegari** [comment]
Thanks

**Pull request merged**